src/functional/iterator/builtins.py

```diff
@@ -109,6 +109,21 @@ def less(*args):
     raise BackendNotSelectedError()
 
 
+@builtin_dispatch
+def lesseq(*args):
```

Comment:
```suggestion
def less_equal(*args):
```
Let's stick to snake case. Same below.